File: pgmeascommons/src/main/java/it/csi/pgmeas/commons/model/Organo.java
/*
* SPDX-FileCopyrightText: (C) Copyright 2025 Regione Piemonte
*
* SPDX-License-Identifier: EUPL-1.2 
*/
package it.csi.pgmeas.commons.model;

import java.io.Serializable;

import it.csi.pgmeas.commons.model.base.DecodificaBaseEntity;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name="pgmeas_d_organo")
public class Organo extends DecodificaBaseEntity implements Serializable {

	private static final long serialVersionUID = 5330415889702797826L;

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "organo_id")
	private Integer organoId;
	
	@Column(name = "organo_cod")
	private String organoCod;
	
	@Column(name = "organo_desc")
	private String organoDesc;
	
	
}

File: pgmeasapp/projects/pgmeas-library/model/src/organo.ts
/*
* SPDX-FileCopyrightText: (C) Copyright 2025 Regione Piemonte
*
* SPDX-License-Identifier: EUPL-1.2 
*/

import { Tipologica } from "./tipologica";

export interface Organo extends Tipologica {
  organoId: number;
  organoCod: string;
  organoDesc: string;
}
